playbooks/technology_radar/artsy-tech-radar.csv

```diff
@@ -1,80 +1,82 @@
+OpenAI,adopt,tools,TRUE, "OpenAI's suite of LLM services and tools for ai assisted development and application features"
+OpenSearch,adopt,tools,FALSE,A community-driven, open source search and analytics suite derived from Elasticsearch
+OpsGenie,hold,tools,TRUE,For incident management and alerting. Being retired by Atlassian.
+Partner API as a product,adopt,techniques,FALSE, "For our business partners to integrate with our platform. Documented at developers.artsy.net"
+Phoenix,hold,languages & frameworks,TRUE,Phoenix is a web development framework written in Elixir which implements the server-side Model View Controller (MVC) pattern.
+Pingdom,hold,tools,TRUE,
+Postgresql,adopt,tools,FALSE,"We rely on Heroku or Amazon's RDS to host these databases. Though MongoDB was chosen to host core API data, in the long run we found the data (and maybe all data) to be more relational than document-oriented."
+Public API as a product,hold,techniques,FALSE,For external developers to build on our platform. Deprecated with plan to retire. Documented at developers.artsy.net
 Python,adopt,languages & frameworks,FALSE,"For ETL, Analytics and ML workloads we use Python as a data engineering standard."
+RabbitMQ,adopt,tools,FALSE,"Its stream allows source systems to publish their notable events and other systems to subscribe as desired. This avoids coupling systems directly. After evaluating Kafka for this purpose, we had more operational success with RabbitMQ."
 Rails,adopt,languages & frameworks,FALSE,
 React Native,adopt,languages & frameworks,FALSE,"New iOS work should use React Native. See also Objective-C."
 React.js,adopt,languages & frameworks,FALSE,
+Redis,adopt,platforms,FALSE,
+Redux,hold,languages & frameworks,FALSE,
 Relay,adopt,languages & frameworks,FALSE,
+Rsbuild,adopt,tools,FALSE,a build tool powered by Rspack. It aims for high performance and to provides an out-of-the-box setup to improve DX.
 Ruby,adopt,languages & frameworks,FALSE,"Especially for data-backed APIs and services"
+Scala,hold,languages & frameworks,FALSE,
+Semaphore,hold,tools,FALSE,
+Serverless architecture,hold,techniques,TRUE,
+Sidekiq,adopt,tools,FALSE,
+SiteFeatures (Gravity),hold,tools,TRUE,"Site Features were used to turn on or off core functionality, but are easily replaced by Unleash (for platform-wide toggles) or Flipper (for Gravity-only settings)."
 Spark,adopt,languages & frameworks,FALSE,"Used for data workloads, mostly via AWS EMR Serverless."
+Split testing,adopt,techniques,TRUE,"When scale permits (such as around high-volume product surfaces or early funnel steps), we prefer to make data-driven product decisions by evaluating user experiences in parallel through A/B- or A/B/*-style tests"
 SQL,adopt,languages & frameworks,FALSE,
-styled-components,adopt,languages & frameworks,FALSE,
+Standard Ruby,trial,tools,TRUE,"After a few weeks living with this officially adopt."
+Styled-components,hold,languages & frameworks,FALSE,
```

Comment:
One caveat here is that Palette is built on top of styled-components! So this technology isn't going anywhere in artsy's systems. It is stable, and stable is good; and it is proven, because of what we've been able to accomplish at scale thanks to its design (including styled-system). And it is certainly fast enough, looking at our core web vitals. 

That said, at some point we need to reckon with the fact that the industry has moved on from this runtime-js based CSS paradigm. This fact is unfortunately too large to really comprehend without an emergency though, given our utter dependency on Palette.